pyaedt/modeler/stackup_3d.py

```diff
@@ -910,6 +916,44 @@ def __init__(self, padstack, layer_name, elevation, thickness):
         self._antipad_radius = 2
         self._units = "mm"
 
+    @property
+    def layer_name(self):
+        """Get the padstack instance layer.
+
+        Returns
+        -------
+        str
+        """
+        return self._layer_name
+
+    @property
+    def pad_radius(self):
+        """Get/Set the pad layer on the specified layer.
```

Comment:
```suggestion
        """Get/Set the pad radius on the specified layer.
```

Reply:
When training me, Alex said that properties should be a noun-based description, with no need for "Get" or "Set". If something unusual existed, an example could be included.

Reply:
good, I'll fix it